src/pages/HomePage.tsx

```diff
@@ -25,19 +25,27 @@ export default function HomePage() {
   const topFiveCoins = data.slice(-5)
 
   return (
-    <div className="home">
-      <Carousel>
-        {topFiveCoins.map((crypto: ICryptoCoin) => (
-          <Card
-            key={crypto.id}
-            border=" border border-purple"
-            title={crypto.name}
-            icon={crypto.image}
-            price={crypto.current_price}
-          />
-        ))}
-      </Carousel>
-      <Table data={data} />
-    </div>
+    <Grid container>
+      <Grid item xs={0} sm={1} md={2} />
```

Comment:
Not sure what you want to achieve with this, but empty Grid is probably not the way to do it.